packages/sdk/src/services/EventServiceByJsonRPC.ts

```diff
@@ -0,0 +1,83 @@
+import axios from 'axios';
+
+export interface BlockResult {
```

Comment:
CasperServiceByJsonRPC return these field in snake case, I think we should unify it and given user only camel case result.

Reply:
Ok